.github/workflows/backport-active.yml

```
@@ -0,0 +1,24 @@
+name: Automatic Backport
+
+on:
+  pull_request:
```

`pull_request` does not grant permissions with `write`, hence you need to use `pull_request_target` instead.

I'm not a big fan of using `pull_request_target`, but we do not use `checkout` or anything along those lines, hence we are safe to use it

```suggestion
  pull_request_target:
```